dotCMS/src/curl-test/CreateAdvanceTemplateOnSpecificHost.postman_collection.json

```diff
@@ -0,0 +1,231 @@
+{
+	"info": {
```

Comment:
Let's move this test to the **TemplateResource collection** that already exists.

Reply:
I prefer to keep it separated